GitSocial/GitSocial/GSCustomLabel.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  GSCustomLabel.swift
+//  GitSocial
+//
+//  Created by Dillon on 11/28/21.
+//
+
+import UIKit
+
+// MARK: - Custom Title Label (No Dynamic Font)
```

Comment:
Thanks for clarifying no dynamic font use here 👍 .. but let's plan to convert this to documentation asap, so it's visible in Xcode. We should plan to implement custom extensions to adapt all labels to dynamic font sizes in future releases.
```suggestion
// MARK: - Custom Title Label (No Dynamic Font)
/// Custom title label that is initialized with a text alignment and font size property.
/// - Does not use Dynamic Font sizing
///     - Ensure font size is large enough when creating title label in app
```

Reply:
Opened issue #6